1_Mechanics/02_GMGuide.md

```diff
@@ -1,82 +1,263 @@
+to reshuffle. If an NPC were to need a mechanical reason to show them being
+exhausted, the same Status Condition can be added to that NPC as if they had
+a level of fatigue.  
+
+### Managing Player Resources
+
+As the Game Master, you can design Adventures at your Table to test different
+aspects of the Players and characters. Each Player holds a single deck of 52
+cards, with two Fate Cards reserved at the beginning of play. Each character
+has statistics that include total Health Points (HP), Fate Points (FP), and
+Armor Points (AP). 
+
+You can think of a Player's deck of cards like their character's stamina. The
+Adventurers should get progressively more tired throughout the day. This is
+accomplished by having the Players make a series of Checks, which is their
+way of interacting with the world. At lower levels, it is OK for Players not
+to approach the re-draw limit of running out of cards between Rests. However,
+as players progress and level up, the Powers they choose and the Encounters
+they face may test them closer to running out of cards between Rests. 
+
+### Resting
+
+HP and FP are also balanced by Rests. It is expected and recommended that a
+party will take 1-2 Quick Rests between Full Rests during each day of in-game
+time, depending on the Encounters they face. While Resting, the Players will
+be regaining their Health and Fate Points, expending Rest Cards along the
+way. This should be kept in mind when planning Encounters that deal damage,
+since Players will want to take a Quick Rest if their HP or FP drops below a
+threshold they feel comfortable with. 
+
+To take a Rest, the Players consult with the Game Master whether it is
+feasible for their characters to secure a 30-minute breather for a Quick
+Rest, or a comfortable, uninterrupted full night's sleep with a Full Rest. If
+a situation is tense, it may always be possible to take a Rest, and it is
+ultimately the Game Master's decision whether a Rest is successfully
+completed or not. In the event that a Rest is interrupted, it can be resumed
+based on the remaining time left and extended as necessary. 
+
+For example, if a group of Adventurers were attempting to spend the night in a
+comfortable inn near the heart of a major city, a Full Rest would be easy to
+secure. If that same group of Adventurers had just snuck into the heart of a
+bandit camp, been spotted, and run into a tent to hide, it would be more
+difficult to secure that Rest. 
+
+A Game Master plays the ultimate authority in the successful completion of a
+Rest, and can ask the Players for a series of Checks to determine whether the
+Rest can be completed or if another factor gets in the way. Players are
+encouraged to work with their Game Master and with each other to preserve
+resources in tense situations where Rests may not be easily acheived.       
+
+### Working with Character Vulnerabilities
+
+When Players create a character, they choose one or two (or more)
```

Comment:
I thought the limit was 2 Vulnys. Or is `or more` directed at the possibility of taking more without XP reward?

Reply:
Yea, I put that in there since Players can technically add as many Vulnys as they want. Technically it's up to 3 since they can take 1 Major and 2 Minor, or even 4 if they take 4 Minor, so I figured leaving it open would be best.